Automattic/wp-calypso
# AnchorFM: Customize Login Text

#### Changes proposed in this Pull Request

Customizes the login text for the AnchorFM onboarding flow

Note: I created a HOC because `Login` is a Class Component, so I couldn't use the `useIsAnchorFm` hook. When trying to use the `useIsAnchorFm` hook in my HOC, Calypso gave me an error (`window.AppBoot is not a function` ) in which I could not decipher the root cause. Perhaps it's because the URL is encoded? Ideally, I could remove all the duplicative code and just use the hook instead.

#### Testing instructions

1. Checkout this branch or use Calypso Live
2. Go to http://calypso.localhost:3000/new?anchor_podcast={anchor_podcast_id}
3. Click "Login"
4. The login subtitle should contain AnchorFM specific text. 

#### Screenshots
<img width="784" alt="Screen Shot 2021-01-13 at 10 46 55 PM" src="https://user-images.githubusercontent.com/66652282/104554765-b28a4580-560a-11eb-998e-1539e4b8de08.png">


Fixes 358-gh-Automattic/dotcom-manage
